README.adoc

```diff
@@ -1,6 +1,15 @@
+== OpenShift.io Services
 
-= Fabric8Analytics Language Server integration
+Red Hat OpenShift.io services extension allows developers using more traditional development tools to benefit from the security, license analysis and AI based suggested dependencies that [OpenShift.io](https://openshift.io/) provides:
+- Common Vulnerabilities and Exposures (CVE) analysis highlights code dependencies that have open CVEs against them.
+- License analysis determines the license used by the project and its dependencies. If the project already has a license the analysis will show any conflicts between that license and dependency licenses that may be more restrictive. If the project does not have a license, OpenShift.io will suggest a license for the project that is sufficiently permissive to match the license restrictions of the dependencies.
+- AI based suggested dependencies that can be added to your application stack, alternative dependencies to currently used dependencies if any of the currently used dependencies are not typically used together in an application stack
 
+Today OpenShift.io services extension supports projects using Maven and projects build on npm (Node ecosystem). Extending support for Python and Go languages is currently under progress.
```

Comment:
is NPM/node supported? and are we going to extend it to python and go

Reply:
It doesn't, but this statement is regarding Openshift.io service and not directed to Fabric8-analytics-devstudio plugin. So kept it as it is.

Reply:
i think otherwise @sivaavkd can update